velocidapter/build.gradle

```diff
@@ -16,7 +16,7 @@ sourceSets {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    implementation 'com.bleacherreport:velocidapter-annotations:1.2.0-beta'
+    api 'com.bleacherreport:velocidapter-annotations:1.2.0-beta'
```

Comment:
This shouldn't be changed, only `velocidapter-android` needs to transitively provide it.